thyrel-web/src/components/home/ButtonModalJoin.tsx

```diff
@@ -27,10 +28,16 @@ export default function ButtonModalJoin({
   return (
     <>
       <BigButton
+        startIcon={
+          !urlIdentifier ? undefined : (
+            <PlayArrowIcon style={{ fontSize: 32 }} />
+          )
+        }
```

Comment:
pas très jolie de tester l'inégalité
```suggestion
        startIcon={
          urlIdentifier ? <PlayArrowIcon style={{ fontSize: 32 }} /> : undefined
        }
```